scripts/databricks_install.py

```diff
@@ -25,7 +26,7 @@
 from databricks_cli.libraries.api import LibrariesApi
 from databricks_cli.dbfs.dbfs_path import DbfsPath
 
-from requests.exceptions import HTTPError
+from generate_conda_file import PIP_BASE
```

Comment:
@anargyri are you sure about `from generate_conda_file import PIP_BASE` vs `from scripts.generate_conda_file import PIP_BASE`? in the setup we are telling users to execute from the root, while the former approach work?

Reply:
I did exactly this
`python scripts/databricks_install.py <CLUSTER_ID>`
and it failed.

Reply:
Which makes sense.

Reply:
we are getting an error:
```
tests/unit/test_notebooks_pyspark.py .....F                              [100%]

=================================== FAILURES ===================================
______________________ test_mmlspark_lightgbm_criteo_runs ______________________

notebooks = {'als_deep_dive': '/data/home/recocat/cicd/4/s/notebooks/02_model/als_deep_dive.ipynb', 'als_pyspark': '/data/home/rec...baseline_deep_dive.ipynb', 'data_split': '/data/home/recocat/cicd/4/s/notebooks/01_prepare_data/data_split.ipynb', ...}

    @pytest.mark.notebooks
    @pytest.mark.spark
    def test_mmlspark_lightgbm_criteo_runs(notebooks):
        notebook_path = notebooks["mmlspark_lightgbm_criteo"]
        pm.execute_notebook(
            notebook_path,
            OUTPUT_NOTEBOOK,
            kernel_name=KERNEL_NAME,
            parameters=dict(
                DATA_SIZE="sample",
                NUM_ITERATIONS=10,
>               EARLY_STOPPING_ROUND=2,
            )
        )

tests/unit/test_notebooks_pyspark.py:65: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/anaconda/envs/reco_pyspark/lib/python3.6/site-packages/papermill/execute.py:94: in execute_notebook
    raise_for_execution_errors(nb, output_path)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

nb = {'cells': [{'cell_type': 'code', 'metadata': {'inputHidden': True, 'hide_input': True}, 'execution_count': None, 'sour...end_time': '2019-03-29T19:50:50.256633', 'duration': 2.430221, 'exception': True}}, 'nbformat': 4, 'nbformat_minor': 2}
output_path = 'output.ipynb'

    def raise_for_execution_errors(nb, output_path):
        """Assigned parameters into the appropriate place in the input notebook
    
        Parameters
        ----------
        nb : NotebookNode
           Executable notebook object
        output_path : str
           Path to write executed notebook
        """
        error = None
        for cell in nb.cells:
            if cell.get("outputs") is None:
                continue
    
            for output in cell.outputs:
                if output.output_type == "error":
                    error = PapermillExecutionError(
                        exec_count=cell.execution_count,
                        source=cell.source,
                        ename=output.ename,
                        evalue=output.evalue,
                        traceback=output.traceback,
                    )
                    break
    
        if error:
            # Write notebook back out with the Error Message at the top of the Notebook.
            error_msg = ERROR_MESSAGE_TEMPLATE % str(error.exec_count)
            error_msg_cell = nbformat.v4.new_code_cell(
                source="%%html\n" + error_msg,
                outputs=[
                    nbformat.v4.new_output(output_type="display_data", data={"text/html": error_msg})
                ],
                metadata={"inputHidden": True, "hide_input": True},
            )
            nb.cells = [error_msg_cell] + nb.cells
            write_ipynb(nb, output_path)
>           raise error
E           papermill.exceptions.PapermillExecutionError: 
E           ---------------------------------------------------------------------------
E           Exception encountered at "In [1]":
E           ---------------------------------------------------------------------------
E           ModuleNotFoundError                       Traceback (most recent call last)
E           <ipython-input-1-1cd6fc35c424> in <module>
E                17 if not is_databricks():
E                18     # get the maven coordinates for MML Spark from databricks_install script
E           ---> 19     from scripts.databricks_install import MMLSPARK_INFO
E                20     packages = [MMLSPARK_INFO["maven"]["coordinates"]]
E                21     repo = MMLSPARK_INFO["maven"].get("repositories", None)
E           
E           /data/home/recocat/cicd/4/s/scripts/databricks_install.py in <module>
E                27 from databricks_cli.dbfs.dbfs_path import DbfsPath
E                28 
E           ---> 29 from generate_conda_file import PIP_BASE
E                30 
E                31 CLUSTER_NOT_FOUND_MSG = """
E           
E           ModuleNotFoundError: No module named 'generate_conda_file'
```
let me try to revert this

Reply:
now it passes

Reply:
We may need to add a `sys.path.append(..)` so that the script works from the CLI.